docs/data/routes/docs/techniques/redirection/headless-ssr/en.md

```diff
@@ -0,0 +1,134 @@
+# Walkthrough: Add Support for Redirects Managed by Content Authors in Headless Server-Side Rendered Applications
+
+To redirect pages based on redirect links provided by Content Authors: 
+
+1. [Create a page template with a redirect field.](#create-a-page-template-with-a-redirect-field) 
+2. [Use the redirect template for new and existing pages.](#use-the-redirect-template-with-new-and-existing-pages)
+3. [Customize the `node-headless-ssr-proxy` sample to handle redirects.](#customize-the-node-headless-ssr-proxy-sample-to-handle-redirects)
+
+## Create a page template with a redirect field
+
+In your Sitecore instance:
+
+1. [Create a data template]((https://doc.sitecore.com/developers/101/sitecore-experience-manager/en/create-a-data-template.html)) called **Redirect Route** that inherits from the `App Route` template.
+2. Under the **Builder** tab, [add a field to the data template](https://doc.sitecore.com/developers/101/sitecore-experience-manager/en/add-a-field-to-a-data-template.html) of type **General Link** field named **Destination** and **Save**. 
+
+## Use the redirect template with new and existing items
+
+1. To create new items with redirect capability, [insert new items in the application's content folder using the new Redirect Route template](https://doc.sitecore.com/users/101/sitecore-experience-platform/en/create-an-item-in-the-content-editor.html#UUID-33faf7af-c5e7-5bc5-5ad9-42325eeb16d2_id__Insert_an_item_1). 
+2. To apply the template to existing items, [edit or change the template that an item is based on](https://doc.sitecore.com/users/101/sitecore-experience-platform/en/edit-or-change-the-template-that-an-item-is-based-on.html).
+
+## Customize the `node-headless-ssr-proxy` sample to handle redirects
+
+> This walkthrough assumes you have followed the ["Headless SSR via `sitecore-jss-proxy`" documentation](/docs/techniques/ssr/headless-mode-ssr) to set up the headless server-side rendering of your application.
+
+1. In the  `/.env` file, define environment variables: 
+
+   ```
+   SITECORE_JSS_APP_NAME="JssReactWeb"
```

Comment:
The env values in this example are specific to my local environment. Also, using `dotenv` together with an `.env` file to specify env variables is one way of doing it, but not the only way. So I think we should define which variables are needed, and then specify that this is an example. Would it be more clear?

Reply:
> So I think we should define which variables are needed, and then specify that this is an example. Would it be more clear?

Yes, definitely.

Reply:
@anastasiya29 Will this do? 
```
 SITECORE_HOSTNAME="<YOUR_JSS_APP_HOSTNAME>"
   SITECORE_GRAPHQL_ENDPOINT="https://<YOUR_CM_HOSTNAME>/api/<YOUR_JSS_APP_NAME>?sc_apikey={<YOUR_API_KEY>}"
   DEFAULT_LANGUAGE="<YOUR_DEFAULT_LANGUAGE>"
```

Reply:
That's clear to me